sqlglot/tokens.py

```diff
@@ -188,6 +188,7 @@ class TokenType(AutoName):
     SHOW = auto()
     SOME = auto()
     STORED = auto()
+    STRUCT = auto()
```

Comment:
if you add this new token type you need to add it to id_var_tokens so that people can use struct as an alias name

select 1 struct